app/cmd/root.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	cfg "github.com/spacemeshos/go-spacemesh/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	config = cfg.DefaultConfig()
+)
+
+// RootCmd is the application root command
+var RootCmd = &cobra.Command{
+	Use:   "spacemesh",
+	Short: "Spacemesh Core ( PoST )",
+}
+
+func init() {
+
+	/** ======================== BaseConfig Flags ========================== **/
+	RootCmd.PersistentFlags().StringVarP(&config.BaseConfig.ConfigFile,
+		"config", "c", config.BaseConfig.ConfigFile, "Set Load configuration from file")
+	RootCmd.PersistentFlags().StringVarP(&config.BaseConfig.DataDir, "datadir", "d",
+		config.BaseConfig.DataDir, "Specify data directory for spacemesh")
+
+	/** ======================== P2P Flags ========================== **/
+	RootCmd.PersistentFlags().IntVar(&config.P2P.SecurityParam, "security-param",
+		config.P2P.SecurityParam, "Consensus protocol k security param")
+	RootCmd.PersistentFlags().IntVar(&config.P2P.TCPPort, "tcp-port",
+		config.P2P.SecurityParam, "TCP Port to listen on")
+	//RootCmd.PersistentFlags().StringVar(&config.P2P.DialTimeout.string, "dial-timeout",
```

Comment:
Why are these commented out ?
also `bootstrap-nodes` and `max-allowed-time-drift`.